app/full_text_search.py

```diff
@@ -281,6 +317,17 @@ def create_romanized_location_fields(romanize_method, **kwargs):
             )
     return fields
 
+def create_non_romanized_fields(**kwargs):
+    """
+    Creates non romanized fields for ranking same kanji higher than
```

Comment:
Now this is not limited to kanji, so you should instead say e.g., "to rank exact matches higher than non-exact matches with the same romanization". Also it would be nice to add example here.